AnagramSolver.Models/Models/Anagram.cs

```diff
@@ -2,7 +2,6 @@
 {
     public class Anagram
```

Comment:
At the moment this class only contains one string, which means this class can be removed and the code can be refactored (after tests cover the area you refactor)